pkgs/applications/networking/instant-messengers/dino/default.nix

```diff
@@ -51,12 +52,19 @@ stdenv.mkDerivation {
     pcre
     xorg.libxcb
     xorg.libpthreadstubs
+    xorg.libXtst
     libXdmcp
     libxkbcommon
     epoxy
     at-spi2-core
     dbus
     icu
+    utillinux
+    libselinux
+    libsepol
+    libpsl
+    brotli
+    libsignal-protocol-c
```

Comment:
These (except `libsignal-protocol-c`) are private dependencies of the actual dependencies that should neither be propagated nor added manually to dino. For example, libpsl and brotli come from libsoup-2.4.pc:
```
Requires: glib-2.0 >=  2.38, gobject-2.0 >=  2.38, gio-2.0 >=  2.38
Requires.private: libxml-2.0, sqlite3, libpsl >=  0.20, libbrotlidec, zlib
```
(To be precise, `glib` uses `utillinuxMinimal` rather than `utillinux`.)

The warnings we see, such as `Package 'mount', required by 'gio-2.0', not found`, come from CMake running both `pkg-config --libs` and `pkg-config --static --libs` to populate both `<XXX>_LIBRARIES` and `<XXX>_STATIC_LIBRARIES` ([docs](https://cmake.org/cmake/help/v3.16/module/FindPkgConfig.html)), but dino has no use for the latter. Currently these warnings can not be disabled: https://gitlab.kitware.com/cmake/cmake/issues/18158

(They could be prevented by pruning `Requires.private` from shared-only libraries akin to #51767, although it can not be detected if a library is shared-only from the .pc file alone, and this is just a warning.)